src/search/parser/errors.h

```diff
@@ -0,0 +1,28 @@
+#ifndef PARSER_ERRORS_H
+#define PARSER_ERRORS_H
+
+#include "../utils/language.h"
+
+#include <string>
+
+// TODO: Remove hack
```

Comment:
What is the hack?

Reply:
I think (github doesn't show me the relevant context) this is related to the `#undef` of a macro with the same name. We already have this macro in the `options` namespace  and copied it here which leads to a duplicated definition. The plan is to get rid of everything in the options namespace and then the `#undef` is no longer needed.